windows-gaming-driver/src/driver/controller.rs

```diff
@@ -158,26 +178,51 @@ impl Controller {
     /// Attaches all configured devices if GA is up and wakes the host up if it's suspended
     pub fn io_attach(&mut self) {
         match self.ga {
-            State::Down | State::Resuming | State::Suspending => (),
+            State::Resuming | State::Suspending => (),
             State::Suspended => {
                 // make them wake up
                 (&self.monitor).send(QmpCommand::SystemWakeup).unwrap();
                 // can't enter now - gotta wait for GA to get ready
                 self.ga = State::Resuming;
             },
-            State::Up | State::Pinging => self.io_force_attach()
+            State::Down => {
+                self.light_attach();
```

Comment:
A notification for the user seems to be approperiate here. 

It is a fallback, so the user ought to know about it